src/pages/GamePage/index.tsx

```diff
@@ -31,7 +30,7 @@ const GamePage = (props: RouteComponentProps) => {
 
   if (error) return <div>Error ${error.message}</div>;
 
-  const { game } = data.store;
+  const { gameBySlug: game } = data.store;
```

Comment:
И тут лучше по старинке, чтобы не сломалось ничего.
```suggestion
  const game = data && data.store && data.store.gameBySlug;
```

Reply:
зачем проверять data?
если это не ошибка и не загрузка значит бэк гарантирует что она есть и определенной структуры
в каких случаях это может сломаться?

Reply:
Да много в каких случаях. Просто это рантайм. И либо так, либо в трай, либо optional chaining прикрутить.

Reply:
ты говоришь о том что при успешном ответе не будет вот такой структуры от бэка?
{
 data: {
   store: {
     gameBySlug: {}
   }
 }
}

когда еще может сломаться?

Reply:
При плохом интернете например, в дате может быть фигня. Ещё в в дате может быть поле error, с объектом ошибки. И это положит все приложение.

Reply:
при плохом интернете будет кейс ошибки
внутри даты error тоже не может быть это тоже кейс ошибки

бэк говорит что это практически не возможный сценарий
звучит не 100%, ну ок поправлю